suave/cmd/suavecli/main.go

```diff
@@ -98,16 +97,16 @@ func unwrapPeekerError(rpcErr error) error {
 	}
 	decodedError, err := hexutil.Decode(rpcErr.Error()[20:])
 	if err != nil {
-		return errors.Join(rpcErr, err)
+		return fmt.Errorf("%s: %s", rpcErr, err)
```

Comment:
hm, so fmt.Errorf accepts a single %w as the error which we could use
something like `fmt.Errorf("unable to decode non-peeker rpc error %w: %s", rpcErr, err.String())`

Reply:
Well, this case both things are errors so it should be maybe `fmt.Errorf("%w, %w)`?